### DISCUSSION TITLE
请教一下，fe存储的元数据主要包含哪些信息

### DISCUSSION THREAD

**Discussion by xiongzegang** [apache/doris#16445] [A - General / Q&A]
请教一下，fe存储的元数据主要包含哪些信息

**Comment by LemonLiTree** [comment]
https://doris.apache.org/zh-CN/docs/dev/admin-manual/maint-monitor/metadata-operation?_highlight=%E5%85%83&_highlight=%E6%95%B0%E6%8D%AE#%E5%85%83%E6%95%B0%E6%8D%AE%E7%9B%AE%E5%BD%95%E7%BB%93%E6%9E%84
